bundles/mapping/drawtools/plugin/DrawPlugin.ol3.js

```diff
@@ -166,14 +179,14 @@ Oskari.clazz.define(
          * @param {String} id
          * @param {boolean} clearCurrent: if true, all selection will be removed from the map
          */
-        stopDrawing : function(id, clearCurrent) {
+        stopDrawing : function(id, clearCurrent, sendEvent) {

```

Comment:
The default should be that event is sent. Use suppressEvent -> if true, don't send event.